packages/settingeditor/src/raweditor.ts

```diff
@@ -49,11 +48,7 @@ export class RawEditor extends SplitPanel {
 
     // Create read-only defaults editor.
     const defaults = (this._defaults = new CodeEditorWrapper({
-      model: new CodeEditor.Model({
-        sharedModel: createStandaloneCell({
-          cell_type: 'code'
-        })
-      }),
+      model: new CodeEditor.Model(),
```

Comment:
In this use case, when disposing the `CodeEditor.Model` we are not disposing the sharedModel because the `CodeEditor.Model` leaves that responsibility to its child (The class that extends it, DocumentModel or CellModel).

I think we should do the same trick we do in the CellModel:
`this._standaloneModel = typeof options.sharedModel === 'undefined';`